relay-dynamic-config/src/defaults.rs

```diff
@@ -450,6 +453,35 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
                     .when(app_start_condition.clone()),
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/ai.total_tokens.used@none".into(),
+            field: Some("span.measurements.ai_total_tokens_used.value".into()),
+            condition: Some(is_ai.clone()),
+            tags: vec![
+                Tag::with_key("span.op")
+                    .from_field("span.sentry_tags.op")
+                    .always(),
+                Tag::with_key("environment")
+                    .from_field("span.sentry_tags.environment")
+                    .always(),
+                Tag::with_key("release")
+                    .from_field("span.sentry_tags.release")
+                    .always(),
+                Tag::with_key("span.description")
+                    .from_field("span.sentry_tags.description")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("span.domain")
+                    .from_field("span.sentry_tags.domain")
+                    .always(), // already guarded by condition on metric
```

Comment:
I think this is just an empty string right now but perhaps it can be repurposed to store the framework information you need.

But perhaps repurposing a column will bite us later on and we should just use a different column.